src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.js

```diff
@@ -113,19 +112,15 @@ const ActivityThread = ({
         <div className="bcs-ActivityThread" data-testid="activity-thread">
             {children}
 
-            {isRepliesLoading && (
-                <div className="bcs-ActivityThread-loading" data-testid="activity-thread-loading">
```

Comment:
Does the styling for this also need to move to the `ActivityThreadReplies` stylesheet?